app/src/main/res/layout/activity_speaker_list.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:id="@+id/content_root"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  tools:context="net.squanchy.speaker.SpeakerListActivity">
+
+  <android.support.design.widget.AppBarLayout
+    style="@style/Squanchy.Appbar"
+    android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <android.support.v7.widget.Toolbar
+      android:id="@+id/toolbar"
+      style="@style/Squanchy.Toolbar.SpeakerList"
+      android:layout_width="match_parent"
+      android:layout_height="?attr/actionBarSize"
+      app:layout_collapseMode="none" />
+
+  </android.support.design.widget.AppBarLayout>
+
+  <net.squanchy.speaker.view.SpeakersView
+    android:id="@+id/speakersView"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent" />
+
+  <ProgressBar
```

Comment:
I'd put this behind the content, not in front (thich means, above in the layout)